modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java

```diff
@@ -58,6 +59,16 @@ public void testGraphQLGetListTypeEntry() throws Exception {
 	public void testGraphQLGetListTypeEntryNotFound() throws Exception {
 	}
 
+	@Override
+	protected ListTypeEntry randomListTypeEntry() throws Exception {
```

Comment:
I see. I'm sorry, you can leave just like this, I was wrong, the first two methods are public. Overriding would be better since the method is already created.